src/qt/sendcoinsdialog.cpp

```diff
@@ -229,10 +235,9 @@ void SendCoinsDialog::on_sendButton_clicked()
     // prepare transaction for getting txFee earlier
     WalletModelTransaction currentTransaction(recipients);
     WalletModel::SendCoinsReturn prepareStatus;
-    if (model->getOptionsModel()->getCoinControlFeatures()) // coin control enabled
-        prepareStatus = model->prepareTransaction(currentTransaction, CoinControlDialog::coinControl);
-    else
-        prepareStatus = model->prepareTransaction(currentTransaction);
+
+    // always throw in the CoinControl instance, it is also used for internal tx-creation parameters (ex. confirmation target)
+    prepareStatus = model->prepareTransaction(currentTransaction, CoinControlDialog::coinControl);
```

Comment:
I don't think we should do it this way. CoinControlDialog::coinControl should not be used if getCoinControlFeatures is disabled. It may have invalid state.
What about creating a temporary CoinControl object and just changing the confirmation setting in the case of non-coin-control?